webpack.config.js

```diff
@@ -137,7 +137,8 @@ const customizer = {
 	...sharedConfig,
 	entry: {
 		'amp-customize-controls': './assets/src/customizer/amp-customize-controls.js',
-		'amp-customize-preview': './assets/src/customizer/amp-customize-preview.js',
+		'amp-customize-controls-legacy': './assets/src/customizer/amp-customize-controls-legacy.js',
```

Comment:
The `defaultConfig` and `sharedConfig` have been merged, so only `sharedConfig` needs to be used:

```diff
--- webpack.config.js	(revision d534577888faca43a1bb229b91ff317cdf4d3dac)
+++ webpack.config.js	(date 1594232015188)
@@ -270,13 +270,12 @@
 };
 
 const mobileRedirection = {
-	...defaultConfig,
 	...sharedConfig,
 	entry: {
 		'mobile-redirection': './assets/src/mobile-redirection.js',
 	},
 	plugins: [
-		...defaultConfig.plugins,
+		...sharedConfig.plugins,
 		new WebpackBar( {
 			name: 'Mobile Redirection',
 			color: '#f27136',
```

Reply:
Fixed in 825e8e91d